axlearn/common/flash_attention/neuron_attention.py

```diff
@@ -57,8 +60,10 @@ def _mha_forward(query, key, value, causal, softmax_scale):
       import neuronxcc.nki.language as nl
 
       assert (num_heads % 2) == 0 and (num_heads // 2 > 0), f'unexpect num_heads: {num_heads}'
-      attn_output, lse = flash_fwd[batch_size, nl.nc(2) * (num_heads//2)](q, k, v, seed, use_causal_mask=causal, softmax_scale=softmax_scale, mixed_precision=True, dropout_p=0.0)
+      attn_output, lse = flash_fwd[batch_size, vnc(2), num_heads//2](q, k, v, bias, seed, use_causal_mask=causal, softmax_scale=softmax_scale, mixed_precision=True, dropout_p=0.0)
```

Comment:
why nl.nc(2) => vnc(2)?

Reply:
seems like ships passing in the night. Both old and new code was vnc(2) when i pulled and tested. It got changed and was missed in merge conflict resolution. PR to fix in a sec